actions/team_actions.jsx

```diff
@@ -62,15 +62,17 @@ export function removeUserFromTeam(teamId, userId, success, error) {
         (data) => {
             getUser(userId)(dispatch, getState);
             TeamActions.getTeamStats(teamId)(dispatch, getState);
-
-            if (data && success) {
-                success();
+            if (data) {
+                getChannelStats(ChannelStore.getCurrentId())(dispatch, getState);
             } else if (data == null && error) {
                 const serverError = getState().requests.teams.removeUserFromTeam.error;
                 error({id: serverError.server_error_id, ...serverError});
             }
         },
     );
+    if (success) {
+        success();
+    }
```

Comment:
This does need to be in the .then() of `removeUserFromTeam` and should be called if data is set, just not in the .then() `of getChannelStats`. Sorry if my previous post was confusing